### ISSUE TITLE
ebay-listbox-button: focus lost on option selection

### ISSUE THREAD

**Issue by agunaganti** [eBay/ebayui-core#1721]
# Bug Report
URL PIEACC-588

## Description
Actual: upon activating the element first time, the focus is lost and moved to the top of page - to the breadcrumbs

Expected: When listbox option is selected and listbox collapses, the focus is set on the listbox button.

## Workaround
- https://github.com/eBay/ebayui-core/issues/1703 -> Previous request in 9.1.0 didn't resolve , we were seeing new issue as mentioned in description.

- Below are few other workarounds, onChange of option, set the focus to button. But, somehow it's not working.
URL redacted

## Screenshots
![Bug5527193_ebay-mweb-view_item-focus_not_managed_when_listbox_option_selected](https://user-images.githubusercontent.com/110565292/182692675-edaf74cf-10c0-4dfa-821b-1acd62ec5551.png)

**Comment by ianmcburnie** [comment]
@agunaganti Does making a selection cause an instant update to the content below? Or Is there a submit button to enact the changes? If it's an instant update then you should be using the eBay-menu-button component.

Also, please refrain from posting corp urls on this public repo. I will go ahead and remove them.

**Comment by ianmcburnie** [comment]
Having said that, we should still fix this issue. Listbox overlay should always collapse and focus should move to button after a selection is made.

**Comment by agunaganti** [comment]
> @agunaganti Does making a selection cause an instant update to the content below? Or Is there a submit button to enact the changes? If it's an instant update then you should be using the eBay-menu-button component.
> 
> Also, please refrain from posting corp urls on this public repo. I will go ahead and remove them.

Sure will skip corp urls.
Yes, its  a instant update to the content below. We were using ebay-listbox-button.

Will try to use, ebay-menu-button as mentioned.

**Comment by agliga** [comment]
https://github.com/eBay/ebayui-core/issues/1703

This has been fixed already, closing.
